net/natmap/Makefile

```diff
@@ -0,0 +1,40 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=natmap
+PKG_VERSION:=20221111
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/heiher/natmap.git
+PKG_SOURCE_DATE:=2022-11-11
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_MIRROR_HASH:=4359bdddd16d7f97af4ee03f814ea4e649554c30eeed93e5f4fc586151d2476e
+
+PKG_MAINTAINER:=Richard Yu <yurichard3839@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=License
+
+PKG_USE_MIPS16:=0
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/natmap
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=TCP/UDP port mapping tool for full cone NAT
+  URL:=https://github.com/heiher/natmap
+endef
+
+TARGET_CFLAGS+= -DCOMMIT_ID=\"$(PKG_VERSION)\"
```

Comment:
```suggestion
TARGET_CFLAGS+= -DCOMMIT_ID="$(PKG_VERSION)"
```

hummm

Reply:
Here should be `-DCOMMIT_ID=\\\"$(PKG_VERSION)\\\"`